src/core.js

```diff
@@ -202,3 +202,18 @@ export function pan(chart, delta, enabledScales, transition = 'none') {
   call(onPan, [{chart}]);
 }
 
+export function getInitialScaleBounds(chart) {
+  const state = getState(chart);
+  const scaleBounds = {};
+  for (const scaleId of Object.keys(chart.scales)) {
+    const original = state.originalScaleLimits[scaleId];
+    if (!original) {
+      continue;
+    }
+
+    const {min, max} = original;
+    scaleBounds[scaleId] = {min: min.scale, max: max.scale};
+  }
```

Comment:
Maybe the `undefined` values would be valuable too, when nothing is found?

```suggestion
  for (const scaleId of Object.keys(chart.scales)) {
    const {min, max} = state.originalScaleLimits[scaleId] || {min: {}, max: {}};
    scaleBounds[scaleId] = {min: min.scale, max: max.scale};
  }
```

Reply:
I was ignoring that case for now, because I assumed that it shouldn't be possible — if the scale exists on the chart, it should also exist on `originalScaleLimits`, I _think_.  But I'm not certain I understand all the states that this can be in, so maybe there's a weird edge case where this becomes important!